gh-weblog/components/index.jsx

```diff
@@ -1,4 +1,4 @@
-import React from "../lib/vendor/react/react.0.12.min.js";
+import {render, h} from "preact";
```

Comment:
This is the one thing I remember really not liking about preact, so I always imported it as `const { render, h as createElement }`. We have minification for that, don't skimp out on function names =)